build.sh

```diff
@@ -195,12 +195,7 @@ else
   fi
 fi
 
-# uncomment RewriteBase in project's .htaccess file
-# necessary for Drupal sites running on a machine configured using ThinkShout standards
-# see https://github.com/thinkshout/ts_recipes/tree/master/brew-lamp-dev-envt
-# see https://github.com/thinkshout/ts_recipes/blob/master/environment_setup.sh
 cd $DESTINATION
-sed -i '' 's/# RewriteBase \/$/RewriteBase \//g' ./.htaccess
```

Comment:
Curious about this -- does it break our local builds now?

Reply:
It doesn't break local, but it just doesn't do anything useful anymore. It WAS however breaking the circle build for some reason... so... just opted to axe it instead of trying to figure out how to fix it!

Reply:
Ah, these lines must not be needed anymore because we're using NGINX instead of Apache locally eh? Anyway, neat! I'll try this out with Facing History and let you know if I come across any bugs.

Reply:
Yeah! Exactly. Thanks for testing!!